Tasks/PullRequestCodeAnalysis/PRCA/PrcaOrchestrator.ts

```diff
@@ -94,25 +95,40 @@ export class PrcaOrchestrator {
     public postSonarQubeIssuesToPullRequest(sqReportPath: string): Promise<void> {
         this.logger.LogDebug(`SonarQube report path: ${sqReportPath}`);
         if (sqReportPath === undefined || sqReportPath === null) {
-            throw new Error("Make sure a SonarQube enabled build task ran before this step.");
+            return Promise.reject('Make sure a SonarQube-enabled build task ran before this step.');
         }
 
         var allMessages:Message[] = this.sqReportProcessor.FetchCommentsFromReport(sqReportPath);
         var messagesToPost:Message[] = null;
-        return this.PrcaService.getModifiedFilesInPr()
+        return Promise.resolve()
```

Comment:
This looks odd, but I needed an empty promise to start the chain so I could accomplish what happens below.